frigate/http.py

```diff
@@ -576,24 +576,14 @@ def timeline():
 @bp.route("/<camera_name>/<label>/thumbnail.jpg")
 def label_thumbnail(camera_name, label):
     label = unquote(label)
-    if label == "any":
-        event_query = (
-            Event.select()
-            .where(Event.camera == camera_name)
-            .order_by(Event.start_time.desc())
-        )
-    else:
-        event_query = (
-            Event.select()
-            .where(Event.camera == camera_name)
-            .where(Event.label == label)
-            .order_by(Event.start_time.desc())
-        )
+    event_query = Event.select(fn.MAX(Event.id)).where(Event.camera == camera_name)
```

Comment:
```suggestion
    event_query = Event.select(fn.MAX(Event.start_time)).where(Event.camera == camera_name)
```

Event IDs just happen to be sortable because the contain the start time. I think its best to use the start time directly.

Reply:
but we need an id to return. in you suggestion, we'll have to complicate the request by grouping, which will slow it down a lot